website/docs/sdk-reference/dart.md

```diff
@@ -102,6 +104,33 @@ final value = await client.getValue(
 );
 ```
 
+## Anatomy of `getValueDetails()`
+
+`getValueDetails()` is similar to `getValue()` but instead of returning only the evaluated value it gives more detailed information about the evaluation result.
+
```

Comment:
example here would be great to demonstrate what "more detailed" means

Reply:
It's there a few lines below in the usage sample